Sources/NIOHTTP1/HTTPTypes.swift

```diff
@@ -1371,10 +1375,11 @@ public enum HTTPResponseStatus {
     case unsupportedMediaType
     case rangeNotSatisfiable
     case expectationFailed
+    case imaTeapot
+    case unprocessableEntity
 
     // 5xx
     case misdirectedRequest
-    case unprocessableEntity
```

Comment:
why is this removed?

Reply:
It’s not, it’s moved to the 4xx section, which is where it belongs. (It’s 422.)